app/cdap/components/shared/Buttons/PrimaryContainedButton/index.tsx

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright © 2022 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+import React from 'react';
+import Button from '@material-ui/core/Button';
+
+export default function PrimaryContainedButton({ children, onClick, disabled = false, ...props }) {
```

Comment:
The goal of the button components is to constrain the app to a few known appearances and to cover behaviors we might want to set on all buttons (e.g. if we want to `disableRipple`)